SPECS/systemd/systemd.spec

```diff
@@ -1,62 +1,342 @@
+Source8:        systemd-journal-gatewayd.xml
+Source9:        20-yama-ptrace.conf
+Source10:       systemd-udev-trigger-no-reload.conf
+
+Source14:       10-oomd-defaults.conf
+Source15:       10-oomd-per-slice-defaults.conf
+Source16:       10-timeout-abort.conf
+Source17:       10-map-count.conf
+
+Source21:       macros.sysusers
+Source22:       sysusers.attr
+Source23:       sysusers.prov
+Source24:       sysusers.generate-pre.sh
+
+Source25:       98-default-mac-none.link
+
+%if 0
+GIT_DIR=../../src/systemd/.git git format-patch-ab --no-signature -M -N v235..v235-stable
+i=1; for j in 00*patch; do printf "Patch%04d:      %s\n" $i $j; i=$((i+1));done|xclip
+GIT_DIR=../../src/systemd/.git git diffab -M v233..master@{2017-06-15} -- hwdb/[67]* hwdb/parse_hwdb.py >hwdb.patch
+%endif
+
+# Backports of patches from upstream (0000–0499)
+#
+# Any patches which are "in preparation" upstream should be listed here, rather
+# than in the next section. Packit CI will drop any patches in this range before
+# applying upstream pull requests.
+
+# Work-around for dracut issue: run generators directly when we are in initrd
+# https://bugzilla.redhat.com/show_bug.cgi?id=2164404
+# Drop when dracut-060 is available.
+Patch0001:      https://github.com/systemd/systemd/pull/26494.patch
+
+
+# Those are downstream-only patches, but we don't want them in packit builds:
+# https://bugzilla.redhat.com/show_bug.cgi?id=1738828
+Patch0490:      use-bfq-scheduler.patch
+
+# Adjust upstream config to use our shared stack
+Patch0491:      fedora-use-system-auth-in-pam-systemd-user.patch
+
+%ifarch %{ix86} x86_64 aarch64
+%global want_bootloader 1
+%endif
+
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  clang
+BuildRequires:  coreutils
 BuildRequires:  libcap-devel
-BuildRequires:  libgcrypt-devel
+BuildRequires:  libmount-devel
+BuildRequires:  libpwquality-devel
+BuildRequires:  pam-devel
+BuildRequires:  p11-kit-devel
 BuildRequires:  libselinux-devel
-BuildRequires:  libxslt
+BuildRequires:  audit-libs-devel
+%if %{without bootstrap}
+BuildRequires:  libxcrypt-devel
+BuildRequires:  cryptsetup-devel
+%endif
+BuildRequires:  dbus-devel
+BuildRequires:  libacl-devel
+# FIXME - uncomment once dep is in repo
+#BuildRequires:  gobject-introspection-devel
+BuildRequires:  libblkid-devel
+%if %{with xz}
+BuildRequires:  xz-devel
+BuildRequires:  xz
+%endif
+%if %{with lz4}
 BuildRequires:  lz4-devel
-BuildRequires:  mariner-release
-BuildRequires:  meson
-BuildRequires:  pam-devel
-BuildRequires:  perl-XML-Parser
-BuildRequires:  python3-jinja2
-BuildRequires:  python3-pyelftools
+BuildRequires:  lz4
+%endif
+%if %{with bzip2}
+BuildRequires:  bzip2-devel
+%endif
+%if %{with zstd}
+BuildRequires:  libzstd-devel
+%endif
+BuildRequires:  libidn2-devel
+BuildRequires:  libcurl-devel
+BuildRequires:  kmod-devel
+BuildRequires:  elfutils-devel
+BuildRequires:  openssl-devel
+%if %{with gnutls}
+BuildRequires:  gnutls-devel
+%endif
+%if %{undefined azure}
```

Comment:
If we hard-code this macro to a specific value, what's the point of the `%if %{undefined azure}`? Can't we just remove these lines? Also, why do we need the `azure` macro at all in this spec?

Reply:
the 'azure' macro won't be hardcoded, it'll come from the standard location in `macros.dist` from the `azure-release-common` package (just like the `fedora-release-common` package); my priority is currently systemd so i'm updating this before changing that package, and then i'll remove the hardcoded macro from here.